spec/services/password_resets/reset_service_spec.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe PasswordResets::ResetService, type: :service do
+  subject(:reset_service) { described_class }
+
+  describe '#call' do
+    let(:user) { create(:user, password: 'HelloLago!1') }
+    let(:membership) { create(:membership, user:) }
+    let(:password_reset) { create(:password_reset, user: membership.user) }
+    let(:reset_args) do
+      {
+        token: password_reset.token,
+        new_password: 'HelloLago!2',
+      }
+    end
+
+    context 'when the reset password service' do
+      it 'changes the user password' do
```

Comment:
I would check here if new password is valid, probably using `authenticate` method

Reply:
Done ! Good one